src/App.jsx

```diff
@@ -1,11 +1,12 @@
-import { useState } from 'react'
 import './App.css'
+import ArticlesPage from './pages/ArticlesPage'
+import { Routes, Route } from "react-router-dom";
 
 function App() {
   return (
-    <>
-      
-    </>
+    <Routes>
+      <Route path="/articles" element={<ArticlesPage />} />
```

Comment:
we should have something at the "/" path, either a homepage, or you can have the articles on this path but as this is the default path we need to show the users something here